spec/system/candidate_interface/candidate_signing_up_spec.rb

```diff
@@ -78,7 +78,15 @@
     it 'sees the start page' do
       visit candidate_interface_welcome_path(token: 'meow')
 
-      expect(page.current_url).to eq(candidate_interface_start_url)
+      expect(page).to have_current_path(candidate_interface_start_path)
     end
   end
+
+  it 'can take the user back to the start page' do
```

Comment:
as above

Reply:
See https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/259#discussion_r331444408.